tools/gluten-te/buildenv.sh

```diff
@@ -19,6 +19,9 @@ USE_ALI_MAVEN_MIRROR=${USE_ALI_MAVEN_MIRROR:-$DEFAULT_USE_ALI_MAVEN_MIRROR}
 # Set timezone name
 TIMEZONE=${TIMEZONE:-$DEFAULT_TIMEZONE}
 
+# Set operating system
+OPERATING_SYSTEM=${OPERATING_SYSTEM:-$DEFAULT_OPERATING_SYSTEM}
```

Comment:
How about using `OS_IMAGE` as the variable name here

Reply:
Thanks, just fixed it.